From 53dc180e7c01038d0248cd00476583b1bfe0cb5c Mon Sep 17 00:00:00 2001
Message-Id: <53dc180e7c01038d0248cd00476583b1bfe0cb5c.1640684961.git.geliang.tang@suse.com>
From: Niklas Cassel <Niklas.Cassel@wdc.com>
Date: Sat, 10 Apr 2021 20:15:43 +0000
Subject: [PATCH] nvme-pci: remove single trailing whitespace
Git-commit: 53dc180e7c01038d0248cd00476583b1bfe0cb5c
Patch-mainline: v5.13-rc1
References: bsn#51


There is a single trailing whitespace in pci.c.
Since this is just a single whitespace, the chances of this affecting
backports to stable should be quite low, so let's just remove it.

Signed-off-by: Niklas Cassel <niklas.cassel@wdc.com>
Signed-off-by: Christoph Hellwig <hch@lst.de>
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 drivers/nvme/host/pci.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/nvme/host/pci.c b/drivers/nvme/host/pci.c
index b06e685d1250..09d4c5f99fc3 100644
--- a/drivers/nvme/host/pci.c
+++ b/drivers/nvme/host/pci.c
@@ -2172,7 +2172,7 @@ static int nvme_setup_io_queues(struct nvme_dev *dev)
 
 	if (nr_io_queues == 0)
 		return 0;
-	
+
 	clear_bit(NVMEQ_ENABLED, &adminq->flags);
 
 	if (dev->cmb_use_sqes) {
-- 
2.31.1

